.changeset/moody-pianos-return.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-link": major
```

Comment:
Does this have an impact on Perseus?

Reply:
No, I checked in the Perseus repo and they don't use this variant.